localization/english/wc_buildings_l_english.yml

```
@@ -195,3 +195,20 @@
  building_citadel_02_desc:0 "Magical wielders tend to prefer the company of like-minded folk who share their interests. To accommodate this, we have created a number of guild rooms, where groups of aligned casters can gather together without disturbance."
  building_citadel_03:0 "Grand Citadel"
  building_citadel_03_desc:0 "Our citadel has become a place of great knowledge and power. Within it are magical artifacts and spellcasters, capable of lifting our nation to greatness and destroying all who would oppose us."
+
+ building_type_titan_forge_01:0 "Titan Forge"
+ building_type_titan_forge_01_desc:1 "TitanForged are created species by Titans themselves, for them to reproduce their only way is to create new ones. For this purpose is Titan Forge."
```

```suggestion
 building_type_titan_forge_01_desc:1 "From the crust of Azeroth, the Titans crafted numerous army known as the Titanforged. With the absence of the Titans, the Keepers lacked the means to produce more Titanforged. Through the Titan Forge, the Keepers are able to replenish their armies."
```

I reworked this one.